.github/workflows/build.yml

```diff
@@ -1,4 +1,4 @@
-# Created with package:mono_repo v6.3.0
+# Created with package:mono_repo v6.4.1
```

Comment:
Can you use `6.3.0`? I will make a separate PR for updating mono_repo for all the packages.

`dart pub global activate mono_repo 6.3.0`